### PR TITLE
Polish some details

### PR DESCRIPTION
**Pull request by raneq**
Update README.md with "new" variable names 
Disable separate cron for restic check
Add task name and date prompt to std{out,err} logs 

tested locally

### CODE REVIEW

**Review by enricostano — APPROVED** [review]

**Comment by raneq** [comment]
@enricostano Solved the merge conflict by deleting the "remove tmp dir" section from upload script. This is already done on the main script. This was decided in PR #10

File: templates/cron-upload.sh.j2 [hunk 1]
```diff
@@ -15,14 +18,10 @@ restic backup \
...
 restic forget --keep-last 5 --prune
 echo "...done"
 
-echo "
-## Restic check ##
-"
+title "Restic check"
 restic check
-echo "...done"
+echo "...done"
```
Comment (enricostano) [inline templates/cron-upload.sh.j2]: Do you mind adding a `\n` at the end of this line?

Comment (raneq) [reply templates/cron-upload.sh.j2]: done. I will take care of this from here on.

File: templates/cron-prepare.sh.j2 [hunk 2]
```diff
@@ -3,9 +3,22 @@
 # Exit with error code 1 if any command fails
 set -e
 
+# Clone standard output to standard error
+#+ This way we have titles and datetimes in both outputs: err and out
+#+ and therefore error messages appear withing a context.
```
Comment (enricostano) [inline templates/cron-prepare.sh.j2]: `within`

File: templates/cron-upload.sh.j2 [hunk 3]
```diff
@@ -3,9 +3,12 @@
 # Exit with error code 1 if any command fails
 set -e
 
-echo "
-## Restic backup ##
-"
+function title {
+  echo -e "\n## $1 ##" | tee /dev/fd/2;
+  date | tee /dev/fd/2;
```
Comment (enricostano) [inline templates/cron-upload.sh.j2]: This `tee` doesn't need `- a`?

**Pull request merged**